src/interactions/views.py

```diff
@@ -77,6 +84,52 @@ def react_to_page(request, *args, pk, **kwargs):
 
 
 @require_http_methods(["POST"])
+def edit_comment(request, *args, comment_id, **kwargs):
```

Comment:
if you aren't using the `args` or `kwargs` but still want to set `comment_id` as a keyword argument, do can change the function like so:
```suggestion
def edit_comment(request, *, comment_id):
```